deploy-board/deploy_board/templates/configs/config_map.tmpl

```diff
@@ -104,6 +105,22 @@
             $('#resetConfigMapBtnId').removeAttr('disabled');
         });
 
+        $('#mapConfigFormId,#newEntryFormId').on('input', 'textarea', function() {
+            const currentVal = $(this).val();
+            if (currentVal.match(/[\r\n\v]+/g)) {
+                $(this).val(currentVal.replace(/[\r\n\v]+/g, ' '));
+                globalNotificationBanner.warning =
+                    "Multi-line (\\r\\n) is not allowed in script config and thus replaced by SPACE. Also note that line wraping is enabled.";
```

Comment:
I do not recommend we replace \r or \n, I think it should only warn the customer and not replace the newlines by space if we have existing use cases. If a user tries to update that, then discovers they can no longer use multiline, it could cause a huge headache. Do you know which users are using multiline in their script config (database contents)?

If there are no cases, then it seems safe.

Reply:
2 having this problem and they are both addressed.